numba/core/options.py

```diff
@@ -92,6 +92,9 @@ class DefaultOptions:
 
     target_backend = _mapping("target_backend")
 
+    dbg_extend_lifetimes = _mapping("dbg_extend_lifetimes")
+    dbg_optnone = _mapping("dbg_optnone")
```

Comment:
```suggestion
    _dbg_extend_lifetimes = _mapping("dbg_extend_lifetimes")
    _dbg_optnone = _mapping("dbg_optnone")
```
(and refactor to match). Suggest making these options available but not explicitly public on the basis that they really need designing into the `debug=` flag as part of a debugging configuration object for that flag to consume.

Reply:
done in d036b1e65